### ISSUE TITLE
web workers

### ISSUE THREAD

**Issue by sethvincent** [sethvincent/virtual-app#4]
I like what's happening in these examples:
- https://github.com/HenrikJoreteg/feather-app
- https://github.com/HenrikJoreteg/featherweight
- https://github.com/colingourlay/duet

Using web workers to diff virtual dom & dom seems like a good way to keep things fast.

Supporting web workers would mean altering virtual-raf or using a different module, though. Because of that I think this is a low priority for now, but still definitely worth exploring at some point.

**Comment by yoshuawuyts** [comment]
yeah, this is a cool idea. See also https://github.com/colingourlay/duet (uses `virtual-dom`)

**Comment by sethvincent** [comment]
Oh, cool! Added to list.

**Comment by yoshuawuyts** [comment]
sickest implementation I've seen - actor model in the browser - @orodio had 4 cores ramping up to 100% while keeping the main (UI) thread running smoothly at 60fps. We should adopt the shit out of this https://github.com/orodio/workers_0002

**Comment by yoshuawuyts** [comment]
cc/ @colingourlay check out James' project - it's so sick

**Comment by colingourlay** [comment]
@yoshuawuyts ahhhhh, @orodio, one of my favourite JS mad scientists
